modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcUtils.java

```diff
@@ -33,7 +33,7 @@
      * @param writer Binary writer.
      * @param items Query results items.
      */
-    public static void writeItems(BinaryWriterExImpl writer, List<List<Object>> items) {
+    public static void writeItems(BinaryWriterExImpl writer, List<List<Object>> items, JdbcProtocolContext protoCtx) {
```

Comment:
why do we add `protoCtx` here and don't use it? 
also, please add to javadoc

Reply:
You right.
This change may be applied when it is needed.
Reverted.